README.md

```diff
@@ -11,6 +11,24 @@ It depends on the [LS Server Core](https://github.com/NYPL-Simplified/server_cor
 * [How to set up a development environment](https://github.com/NYPL-Simplified/Simplified/wiki/Deployment-Instructions)
 * Two sets of Ansible playbooks are in development: [One developed by Minitex](https://github.com/Minitex/ansible-playbook-libsimple) and [a derivative developed by Amigos Library Services](https://github.com/alsrlw/ansible-playbook-libsimple)
 
+## Generating Documenation
+
+Code documentation using Sphinx can be found on [readthedocs.org](). It currently documents this repo's `api` directory, `scripts` file, and the `core` directory submodule. The configuration for the documentation can be found in `/docs`.
+
+There are two steps to building the documentation: generate the `.rst` files from the codebase and generate the html static site (not to be committed). Inside the `/docs` directory, run:
```

Comment:
Why are the `.rst` files committed?

Reply:
Readthedocs doesn't generate them, it only builds the static website from those files.

Reply:
I'm unhappy about this. I'd like to think about ways to avoid keeping these files in the repo, or at least putting the CI process in charge of keeping them updated.